### PR TITLE
Add test case for EI-780

### PR DESCRIPTION
**Pull request by Subasinghe**
This test class is to check the presence of MIMEBoundary value in Content-Type header when converting JSON payload to multipart/form-data 
issue - #780

### CODE REVIEW

File: integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/nhttp/transport/MultipartFormdataMIMEBoundaryTest.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    </registry>
+    <api xmlns="http://ws.apache.org/ns/synapse"
+         name="MIMEBoundaryTestAPI"
+         context="/testMIMEBoundary">
+        <resource methods="POST">
+            <inSequence>
+                <property name="messageType" value="multipart/form-data" scope="axis2"/>
+                <respond/>
+            </inSequence>
+            <outSequence>
```
Comment (VijithaEkanayake) [inline integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/nhttp/transport/MultipartFormdataMIMEBoundaryTest.xml]: Do we need outSequence in synapse configuration?
